src/newViews/ProjectEdit/GeoAreas/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import {
+    Button,
+    ListView,
+    useBooleanState,
+} from '@the-deep/deep-ui';
+import {
+    IoAdd,
+    IoChevronUpOutline,
+    IoChevronDownOutline,
+} from 'react-icons/io5';
+
+import {
+    BasicRegion,
+} from '#typings';
+import _ts from '#ts';
+
+import RegionCard from './RegionCard';
+import styles from './styles.scss';
+
+const regionKeySelector = (d: BasicRegion): number => d.id;
+
+interface Props {
+    className?: string;
+    regions: BasicRegion[];
+}
+
+function GeoAreas(props: Props) {
+    const {
+        className,
+        regions,
+    } = props;
+
+    const regionRendererParams = useCallback((_: number, data: BasicRegion) => ({
+        region: data,
+    }), []);
+
+    const [
+        showContent,,,,
+        toggleContentVisibility,
+    ] = useBooleanState(false);
+
+    return (
+        <div className={_cs(className, styles.geoAreas)}>
+            <div className={styles.mapContainer}>
+                map
+            </div>
+            <div className={styles.listContainer}>
+                <Button
+                    className={styles.addCustom}
+                    variant="secondary"
+                    name="addCustomGeo"
+                    icons={<IoAdd />}
+                >
+                    {_ts('geoAreas', 'addCustom')}
+                </Button>
+                <div className={styles.geoAreasDropdown}>
+                    {_ts('geoAreas', 'title')}
+                    <Button
+                        className={styles.button}
+                        name="dropdown"
+                        variant="action"
+                        disabled={regions.length <= 0}
+                        actions={showContent ? (
+                            <IoChevronUpOutline />
+                        ) : (
+                            <IoChevronDownOutline />
+                        )}
+                        onClick={toggleContentVisibility}
+                    />
+                </div>
+                {showContent && (
+                    <ListView
+                        className={styles.regions}
+                        data={regions}
+                        rendererParams={regionRendererParams}
+                        renderer={RegionCard}
+                        rendererClassName={styles.region}
+                        keySelector={regionKeySelector}
+                    />
+                )}
```

Comment:
We already have a component to do this.
Also, the default state should not be collapsed right @AdityaKhatri ?

Reply:
I have made the necessary changes